packages/components/src/tripInput/__tests__/TripInput.test.js

```diff
@@ -12,7 +12,10 @@ it('renders', () => {
     shallow(
       <TripInput
         onPress={jest.fn()}
-        icon={<IconMock />}
+        icon={
+          // $FlowExpectedError: TripInput expects Icon component
+          <IconMock />
```

Comment:
Cannot we use `Icon` component as `IconMock` instead of `null` to get rid of all `FlowExpectedError`?

Reply:
I guess the idea behind using this mock is to avoid refactoring this snapshot test if the `Icon` component's props change.
@tbergq Could you provide more context?

Reply:
@RobinCsl Thanks, that makes sense.